__tests__/app.test.js

```diff
@@ -49,33 +49,70 @@ describe("GET /api/topics", () => {
   });
 });
 
-describe("GET /api/articles/:article_id", () => {
-  test("200: returns a single article with the required properties", () => {
-    return request(app)
-      .get("/api/articles/1")
-      .expect(200)
-      .then(({ body }) => {
-        const { article } = body;
-        expect(article).toMatchObject({
-          author: expect.any(String),
-          title: expect.any(String),
-          article_id: expect.any(Number),
-          body: expect.any(String),
-          topic: expect.any(String),
-          created_at: expect.any(String),
-          votes: expect.any(Number),
-          article_img_url: expect.any(String),
+describe("/api/articles/:article_id", () => {
+  describe("GET", () => {
+    test("200: returns a single article with the required properties", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({ body }) => {
+          const { article } = body;
+          expect(article).toMatchObject({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            body: expect.any(String),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+          });
         });
-      });
-  });
+    });
 
-  test("GET:400 sends an appropriate status and error message when given an invalid id", () => {
-    return request(app)
-      .get("/api/articles/article")
-      .expect(400)
-      .then(({ body }) => {
-        expect(body).toEqual({ message: "Bad request" });
-      });
+    test("400: sends an appropriate status and error message when given an invalid id", () => {
+      return request(app)
+        .get("/api/articles/article")
+        .expect(400)
+        .then(({ body }) => {
+          expect(body).toEqual({ message: "Bad request" });
+        });
+    });
+  });
+  describe("PATCH", () => {
+    test("200: increments the articles votes by the specified amount", () => {
+      return request(app)
+        .patch("/api/articles/1")
+        .send({ inc_votes: 1 })
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.article).toMatchObject({
+            article_id: 1,
+            votes: 101,
+          });
```

Comment:
nice test here, not necessary to check for any other properties here. you've checked the two main ones!